### ISSUE TITLE
Questions to the REOF power parameter in the examples

### ISSUE THREAD

**Issue by SHEN-Cheng** [xarray-contrib/xeofs#168]
I feel lost of why the power set to 4 here:
```py
# (3) Promax-rotated EOF analysis
rot_pro = EOFRotator(n_modes=50, power=4)
```
I understand the power set 1 can return a Varimax-rotated EOF result, but why 4 can get a Promax-rotated EOF result in this example?

Sorry, this question might beyond the respositry.

**Comment by SHEN-Cheng** [comment]
> No worries! Did you have a look into the review paper of [Richmann (1986)](http://www.atmo.arizona.edu/students/courselinks/fall09/atmo529/Lectures/Richman1986.pdf)? In section 8.5 he describes the Promax procedure. In particular, he defines the elements of target matrix of the Procrustes transformation as
> 
> bij=|aijm+1|/aij
> 
> where aij are the elements of the matrix obtained by Varimax rotation. For m=1, you simply get bij=aij, i.e. the Varimax solution. For m≥1, the bij will differ from aij and, in general, the obtained matrix won't be orthogonal anymore.
> 
> In xeofs, the power parameter p of the `EOFRotator` class represents m. Does that clear up the confusion?

Thanks for the quickreply, I haven't looked into that paper. I will have a look of it soon. BTW, a quick question: what is the difference of m=3 and m=4 if I want to get a Promax REOF?

**Comment by nicrie** [comment]
There is not a single Promax rotation, but indefinitely many: it depends on $m$. You can choose a "weak" or "strong" rotation, with larger mm resulting in stronger rotations. In a nutshell, the Promax power parameter $m$ enhances the effect of driving smaller eigenvector values even more towards 0 by raising the values to the power $m$. The larger $m$, the more you will discriminate lower values from larger values.

### An Example

Take two eigenvector values, 0.4 and 0.6, obtained from Varimax rotation. Raise both to the power of $m=2$: 0.16 and 0.36. Alternatively, raise both to the power of, say, $m=5$: 0.01 and 0.08.

In the Varimax solution, the second value was 50% larger than the first value. With Promax $m=2$, the second value is 125% larger. And with Promax $m=5$, the second value is 700% larger. So the power parameter effectively controls the relative importance of individual values.

In summary, higher values of the Promax power parameter $m$ will result in solutions that focus more on the important eigenvector values, at the cost of introducing more correlation between the principal component (PC) scores, which are uncorrelated for Varimax.

**Comment by SHEN-Cheng** [comment]
> There is not a single Promax rotation, but indefinitely many: it depends on m. You can choose a "weak" or "strong" rotation, with larger m resulting in stronger rotations. In a nutshell, the Promax power parameter m enhances the effect of driving smaller eigenvector values even more towards 0 by raising the values to the power m. The larger m, the more you will discriminate lower values from larger values.
> 
> ### An Example
> Take two eigenvector values, 0.4 and 0.6, obtained from Varimax rotation. Raise both to the power of m=2: 0.16 and 0.36. Alternatively, raise both to the power of, say, m=5: 0.01 and 0.08.
> 
> In the Varimax solution, the second value was 50% larger than the first value. With Promax m=2, the second value is 125% larger. And with Promax m=5, the second value is 700% larger. So the power parameter effectively controls the relative importance of individual values.
> 
> In summary, higher values of the Promax power parameter m will result in solutions that focus more on the important eigenvector values, at the cost of introducing more correlation between the principal component (PC) scores, which are uncorrelated for Varimax.

Many thanks for the knowledege and your time to answer this question. I believe I had an improved understanding on the m impact on the effect of the Rotation.

**Comment by nicrie** [comment]
Closing. Feel free to reopen if you still have questions regarding this @SHEN-Cheng
